src/Uno.UWP/Globalization/CalendarIdentifiers.cs

```diff
@@ -0,0 +1,51 @@
+namespace Windows.Globalization
+{
+	public sealed partial class CalendarIdentifiers 
+	{
+		internal const string JulianValue = "JulianCalendar";
+		public static string Julian => JulianValue;
```

Comment:
any reason not to use a readonly property? Ie `public static string Julian {get; } = "JulianCalendar";`

Reply:
To be able to use them in a `switch`: https://github.com/nventive/Uno/pull/906/files#diff-4a4eda7d031cd847ffbfe3b64d30a063R17